src/Orleans/Serialization/BuiltInTypes.cs

```diff
@@ -1850,6 +1852,7 @@ internal static void SerializeInvokeMethodRequest(object obj, ISerializationCont
         internal static object DeserializeInvokeMethodRequest(Type expected, IDeserializationContext context)
         {
             int iid = context.StreamReader.ReadInt();
+            ushort iVersion = context.StreamReader.ReadUShort();
```

Comment:
We can't tell if there's more data left, but we could narrow argCount to a `ushort` and put `iVersion` where the most significant bits currently are